### DISCUSSION TITLE
Can't change Activity's assigned project after creation

### DISCUSSION THREAD

**Discussion by artisleilands** [kimai/kimai#3253] [General]
So, this happens:

1. An activity is created, assigned to a project, saved
2. When editing the activity, it is not possible to change the assigned project – only an empty space or the already assigned project.
3. When empty space is assigned as the project and activity saved, it is no longer possible to assign any project, the project field disappears.

Actually, it is also not possible to assign a project if the activity is saved without one. There is no field for that.

**Comment by kevinpapst** [comment]
You described the situation correct.

You can read why at: https://www.kimai.org/documentation/activity.html#editing-an-activity

**Reply by artisleilands** [reply]
I see – feature, not a bug. Thank you!

**Reply by kevinpapst** [reply]
I will update the docs to include the workaround:

- Create the new activity 
- Go to the timesheet section for all users
- Filter all timesheet for the old activity (increase the page size if there are more than 50 results)
- Click the checkbox in the upper left of the result table to check all timesheet rows
- Now choose "Edit" from the dropdown below the table for batch-update mode
- In the batch-update screen: choose the project and new activity and save
- After there is no more timesheet with the old activity you can safely delete it
